src/routes/docs/products/databases/queries/+page.markdoc

```diff
@@ -418,3 +418,294 @@ query {
 }
 ```
 {% /multicode %}
+
+# Multiple Queries {% #multiple-queries %}
+
+When working with databases, you often need to combine multiple conditions to filter your data effectively. Appwrite provides several ways to create complex queries by combining conditions using `AND` and `OR` operations.
+
+## Default AND behavior
+When you pass multiple queries to `listDocuments()`, they are automatically combined with `AND` logic. This means a document must match all the conditions to be included in the results. For example, to find movies titled either "Avatar" or "Lord of the Rings" that were released after 1999:
+
+{% multicode %}
+```client-web
+const results = await databases.listDocuments(
+    '<DATABASE_ID>',
+    '<COLLECTION_ID>',
+    [
+        Query.equal('title', ['Avatar', 'Lord of the Rings']),
+        Query.greaterThan('year', 1999)
+    ]
+);
+```
+```client-flutter
+final results = await databases.listDocuments(
+    '<DATABASE_ID>',
+    '<COLLECTION_ID>',
+    [
+        Query.equal('title', ['Avatar', 'Lord of the Rings']),
+        Query.greaterThan('year', 1999)
+    ]
+);
+```
+```python
+results = databases.list_documents(
+    database_id='<DATABASE_ID>',
+    collection_id='<COLLECTION_ID>',
+    queries=[
+        Query.equal('title', ['Avatar', 'Lord of the Rings']),
+        Query.greater_than('year', 1999)
+    ]
+)
+```
+```client-apple
+let results = try await databases.listDocuments(
+    databaseId: '<DATABASE_ID>',
+    collectionId: '<COLLECTION_ID>',
+    queries: [
+        Query.equal("title", value: ["Avatar", "Lord of the Rings"]),
+        Query.greaterThan("year", value: 1999)
+    ]
+)
+```
+```kotlin
+val results = databases.listDocuments(
+    databaseId = '<DATABASE_ID>',
+    collectionId = '<COLLECTION_ID>',
+    queries = listOf(
+        Query.equal("title", listOf("Avatar", "Lord of the Rings")),
+        Query.greaterThan("year", 1999)
+    )
+)
+```
+{% /multicode %}
+
+In this example, the first query uses an array in `equal()` to match either title, creating an `OR` condition within that single query. The second query adds an `AND` condition requiring the year to be after 1999. The database will return documents where the title matches either "Avatar" OR "Lord of the Rings" AND the year is greater than 1999.
```

Comment:
This is repetitive since the same thing is in the Building Queries section.

Reply:
Right, removed!